### PR TITLE
Maven run mojo

### PR DESCRIPTION
**Pull request by MykolaGolubyev**
Fixes https://github.com/twosigma/webtau/issues/99

### CODE REVIEW

File: webtau-groovy/src/main/groovy/com/twosigma/webtau/cli/WebTauCliApp.groovy [hunk 1]
```diff
@@ -114,7 +112,10 @@ class WebTauCliApp implements StandaloneTestListener, ReportGenerator {
     }
 
     private List<Path> testFiles() {
-        return cliConfigHandler.testFiles.collect { Paths.get(it) }
+        return cliConfigHandler.testFiles.collect { fileName ->
+            def path = Paths.get(fileName)
+            return path.isAbsolute() ? path : cfg.workingDir.resolve(path)
```
Comment (tsiq-karold) [inline webtau-groovy/src/main/groovy/com/twosigma/webtau/cli/WebTauCliApp.groovy]: Thinking out loud, not convinced it's a good idea but... is it worth having a helper function on `WebTauConfig` to do this absolute/relative thing?  It's not the only place you do the exact same tertiary check.

File: webtau-maven-plugin-test/pom.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.twosigma.webtau</groupId>
+                <artifactId>webtau-maven-plugin</artifactId>
+                <version>${project.version}</version>
+                <configuration>
+                    <workingDir>${project.basedir}/src/tests/groovy</workingDir>
```
Comment (tsiq-karold) [inline webtau-maven-plugin-test/pom.xml]: So just to make sure I understand this correctly, to use non-default env you'd do `<env>myEnv</env>`?

Comment (MykolaGolubyev) [reply webtau-maven-plugin-test/pom.xml]: yes

File: webtau-groovy/src/main/groovy/com/twosigma/webtau/cli/WebTauCliApp.groovy (hunk 1 above)
Comment (MykolaGolubyev) [reply webtau-groovy/src/main/groovy/com/twosigma/webtau/cli/WebTauCliApp.groovy]: Was thinking the same. Makes sense to me.

**Pull request merged**